tests/test_polydata.py

```diff
@@ -857,3 +857,19 @@ def test_n_verts():
 def test_n_lines():
     mesh = pyvista.Line()
     assert mesh.n_lines == 1
+
+
+def test_geodesic_error_message(sphere, sphere_shifted):
+
```

Comment:
I'd slightly prefer a semantically oriented name, e.g.
```suggestion
def test_geodesic_disconnected(sphere, sphere_shifted):
```
I'd also remove the empty line (github doesn't seem to show this in this edit suggestion for some reason).

Reply:
Yeah that's much clearer